app/views/messages/new.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<%= form_for @message, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
```

Comment:
I think this will default to the create action so you don't need to pass the `:url` option.